app/server/openshift/src/main/java/io/syndesis/server/openshift/OpenShiftServiceImpl.java

```diff
@@ -75,10 +74,21 @@
 
     private final NamespacedOpenShiftClient openShiftClient;
     private final OpenShiftConfigurationProperties config;
+    private Map<String, String> productizedLabelsTemplate = new HashMap<>();
 
     public OpenShiftServiceImpl(NamespacedOpenShiftClient openShiftClient, OpenShiftConfigurationProperties config) {
         this.openShiftClient = openShiftClient;
         this.config = config;
+        // retrieve the productized labels: com.redhat, they exists only when syndesis-operator is built with productized=true
+        DeploymentConfig dc = openShiftClient.deploymentConfigs() != null  ?
+            openShiftClient.deploymentConfigs().withName("syndesis-server").get() : null;
+        if (dc != null) {
+            Map<String, String> labels = dc.getSpec().getTemplate().getMetadata().getLabels();
+            productizedLabelsTemplate = Collections.unmodifiableMap(
+                labels.entrySet().stream()
+                    .filter(entry -> entry.getKey().contains("com.redhat"))
```

Comment:
This doesn't feel right...
The PR is going technically into upstream master yet we are specifically looking for company-centric product-only labels. Is there a way perhaps to add these to a properties file that is only included in product?

@zregvart WDYT?